build.sbt

```diff
@@ -89,6 +91,53 @@ lazy val coreJVM = core.jvm
 lazy val coreNative = core.native
   .settings(nativeSettings)
 
+lazy val coreTests    = crossProject(JSPlatform, JVMPlatform)
+  .in(file("core-tests"))
+  .settings(stdSettings("zio-prelude-tests"))
+  .settings(crossProjectSettings)
+  .settings(macroDefinitionSettings)
+  .settings(buildInfoSettings("zio.prelude.tests"))
+  .settings(Compile / console / scalacOptions ~= { _.filterNot(Set("-Xfatal-warnings")) })
```

Comment:
Why turning off Fatal warning? :thinking:

Reply:
I just copied the existing settings from `core` so I figured you had your reasons. Happy to change. 😃

Reply:
Oh, silly me, that's just for `console` :facepalm:
Please ignore my question above :D